libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-  color: var(--color-white);
-  font-size: 1rem;
-  font-weight: 500;
+  background-color: #efefef;
+  border: 1px solid var(--color-white);
+  border-radius: 0.5rem;
+  cursor: pointer;
+  display: flex;
+  margin: 0 0 16px;
+  padding: 16px 24px;
+  user-select: none;
 }
 
-.quiz-container .foreground .heading-XL {
-  font-size: 2.5rem;
-  line-height: 3.75rem;
-  margin: 0 0 27px;
-  font-weight: 900;
+.quiz-option-icon {
+  display: flex;
+  line-height: 0;
+  margin-right: 16px;
 }
 
-.uar-genericQuiz-container .quiz-content-wrapper .quiz-container .quiz-heading {
-  font-size: 3rem;
-  line-height: 3.75rem;
-  margin: 0 0 27px;
+.quiz-option-icon img {
+  height: var(--icon-size-m);
+  width: var(--icon-size-m);
 }
 
-.quiz-container .consonant-CardsGrid {
+.quiz-option-image {
   display: flex;
-  flex-direction: column;
-}
-
-/* Added to prevent click event triggering below the cards 
-TODO: Need to check if we can skip this somehow.
-*/
-.quiz-container .consonant-CardsGrid--with4xGutter {
-  padding-bottom: 0;
-  gap: 0;
+  height: 46px;
+  margin-right: 16px;
+  object-fit: cover;
+  width: 46px;
 }
 
-.quiz-container .card .consonant-OneHalfCard-title {
+.quiz-option-title {
+  color: var(--text-color);
   display: none;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 22px;
+  margin: 0 0 8px;
+  text-align: center;
 }
 
-div[data-valign="middle"] {
-  align-self: center;
-}
-
-.quiz-container .consonant-OneHalfCard-image {
-  line-height: 0;
-}
-
-.quiz-container .consonant-OneHalfCard-text {
-  font-size: 16px;
-  color: #000;
+.quiz-option-text {
+  color: var(--text-color);
+  font-size: var(--type-body-xs-size);
+  line-height: var(--type-body-xs-lh);
   margin: 0;
+  text-align: center;
 }
 
-.quiz-container .card.selected .consonant-OneHalfCard-text {
-  color: var(--link-hover-color-dark);
-}
-
-.quiz-container .card.selected .consonant-OneHalfCard-text {
-  color: var(--link-hover-color-dark);
-  font-weight: bold;
-}
-
-.quiz-container .card.selected {
+.quiz-option:hover {
   background-color: var(--color-white);
 }
 
-.quiz-container .card.selected .consonant-OneHalfCard-title {
-  font-weight: bold;
-  color: var(--link-hover-color-dark);
-}
-
-.quiz-container .card.selected {
+.quiz-option.selected {
   background-color: var(--color-white);
 }
 
-.quiz-container .consonant-OneHalfCard-inner {
-  padding: 20px;
-}
-
-.quiz-container .consonant-OneHalfCard {
-  background-color: #efefef;
-  border: 1px solid var(--color-white);
-}
-
-.milo-card-wrapper {
-  padding: 20px;
-}
-
-.quiz-container .consonant-CardsGrid--5up .card > div { 
-  width: 100%;
-}
-
-.quiz-container .consonant-CardsGrid--5up .card { 
-  width: 100%;
-  min-width: 100%;
-  max-width: 100%;
-  margin: 0 0 16px;
-}
-
-.quiz-container .consonant-CardsGrid--5up .card div div {
-  text-align: right;
+.quiz-option.selected .quiz-option-title {
+  color: var(--link-hover-color-dark);
+  font-weight: bold;
```

Comment:
Earlier I saw `900` was used as the value for a `font-weight`. Maybe it would be a good idea to follow a pattern of just using number values?